.clangd

```diff
@@ -5,7 +5,6 @@
 Diagnostics:
   Includes:
     IgnoreHeader:
-      - core/typedefs\.h # Our "main" header, featuring transitive includes; allow everywhere.
```

Comment:
I guess you were able to remove this by `IWYU pragma: begin_exports` in the file?

Reply:
Yep! Wasn't aware of this pragma's existence when making the original commit